src/System.Private.ServiceModel/tools/scripts/CleanUpHttpsPort.cmd

```diff
@@ -0,0 +1,34 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+REM
+echo Remove Https configuration
+REM Note that we always return successful code in clean up routine because clean up errors are not fatal
```

Comment:
You should be checking if the script is being run elevated. checking whether or not `net session` exits with errorlevel 0 should be a sufficient indicator of whether or not you're elevated.